geomstats/learning/expectation_maximization.py

```diff
@@ -293,10 +293,31 @@ def fit(self,
             Gaussian mixture model: means, variances and mixture_coefficients.
         """
         self._dimension = data.shape[-1]
-        self.means = (gs.random.rand(
-            self.n_gaussians,
-            self._dimension) - 0.5) / self._dimension
-        self.variances = gs.random.rand(self.n_gaussians) / 10 + 0.8
+        if self.initialisation_method == "kmeans":
+            kmeans = RiemannianKMeans(metric=self.metric,
+                                      n_clusters=self.n_gaussians,
+                                      init='random',
+                                      mean_method='frechet-poincare-ball'
+                                      )
+
+            centroids = kmeans.fit(X=data, max_iter=100)
+            labels = kmeans.predict(X=data)
+
+            self.means = centroids
+            self.variances = gs.zeros(self.n_gaussians)
+
+            labeled_data = gs.vstack([labels, data.T]).T
```

Comment:
NIT: If possible, we try to use `gs.transpose` instead of `T`: this makes the generalization of the code to pytorch and tf easier, later!

Reply:
Ah, I didn't even notice there was a `gs.transpose`. That makes sense; I'll make the change 👍